frontend/public/components/catalog-items.jsx

```diff
@@ -354,9 +366,10 @@ export class CatalogTileViewPage extends React.Component {
               ? this.renderCategoryTiles(activeCategory)
               : _.map(currentCategories, category => (category.numItems && category.id !== 'all') ? this.renderCategoryTiles(category) : null)}
           </CatalogTileView>}
-          {numItems === 0 && <EmptyState className="blank-slate-content-pf">
-            <EmptyState.Title>No Results Match the Filter Criteria</EmptyState.Title>
-            <EmptyState.Info>
+          {numItems === 0 && <EmptyState className="co-catalog-page__no-filter-results">
+            <EmptyState.Title className="co-catalog-page__no-filter-results--title" aria-level="2">
+              No Results Match the Filter Criteria</EmptyState.Title>
```

Comment:
style nit: move closing tag to line below to be consistent with the other elements here